scripts/qa_crowbarsetup.sh

```diff
@@ -3133,6 +3134,20 @@ function set_proposalvars
     fi
 }
 
+function set_noproxyvar
+{
+    [[ $http_proxy ]] || [[ $https_proxy ]] || return
+    no_proxy="$adminip,$no_proxy"
+    export no_proxy="${no_proxy%,}";
```

Comment:
should'nt we want to export it at the very end, when $net_public is also included?